<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vp771</class>
 <widget class="QWidget" name="vp771">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
    <item>
     <widget class="QLabel" name="kvp_status_icon">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="ressources.qrc">:/icons/img/ko.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblkvp">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>SCALER KRAMER VP-771</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="kvp_blank">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>100</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>BLANK</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="kvp_freeze">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>100</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>FREEZE</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="kvp_pattern">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>367</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Mire (Off)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Slide Bar (non-HDCP)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Color Bar (HDCP)</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="kvp_hdcpFollow">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>244</y>
     <width>125</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Follow Output</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Follow Input</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="kvp_hdcp">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>247</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>HDCP</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="kvp_inputs">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>211</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="kvp_cv1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>CV1</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="kvp_hdmi2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>HDMI2</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="kvp_vga2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>VGA2</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="kvp_rgb">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>RGB</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="kvp_vga1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>VGA1</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="kvp_hmdi1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>HDMI1</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="kvp_cv2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>CV2</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="kvp_sdi">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>SDI</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="kvp_dp">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>DP</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>211</width>
     <height>438</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(251, 255, 210);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>kvp_blank</zorder>
  <zorder>kvp_freeze</zorder>
  <zorder>kvp_pattern</zorder>
  <zorder>kvp_hdcpFollow</zorder>
  <zorder>kvp_hdcp</zorder>
  <zorder>kvp_inputs</zorder>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
